public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
+
 There are many Ethereum clients to choose from. We recommend using different clients when developing and deploying.
 
-# When Developing
+## Developing locally
+
+### Truffle Develop
+
+We recommend using Truffle Develop, an Ethereum client built directly into Truffle which contains an in-memory blockchain that runs only on your development machine.
+
+It processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly, and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing.
+
+Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following in a terminal:
+
+```shell
+truffle develop
+```
+
+This will run the client on `http://localhost:9545`. it will display the first 10 accounts, and a deterministic mnemonic for the blockchain. Truffle Develop uses the same mnemonic upon every launch:
+
+```
+candy maple cake sugar pudding cream honey rich smooth crumble sweet treat
+```
+
+Once the Truffle Develop console is launched, you can run all of the commands you are used to, but omitting the `truffle` prefix. So, for example, to compile your smart contract, instead of typing `truffle compile` on the terminal, just type `compile` in the Truffle Develop console.
+
+### TestRPC
+
+You can also use [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). Like Truffle Develop, it is an in-memory blockchain that runs only on your development machine. Truffle knows how to use its special features to speed up test runtime by almost 90%.
+
+To run the TestRPC, open a new terminal and type:
+
+```shell
+testrpc
+```
 
-* EthereumJS TestRPC: [https://github.com/ethereumjs/testrpc](https://github.com/ethereumjs/testrpc)
+The TestRPC will start with a different seed each time (and so a different mnemonic and account list), but you can run the TestRPC with a specific mnemonic in order to regenerate the same accounts:
```

Comment:
As mentioned above, this is a pretty good sell. Also, providing an example of how to use the TestRPC here emphasizes the TestRPC instead of deemphasizing it. We should like pare it down to something like this:

> You can also use the [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). It's a separate command-line blockchain you can use for both development and automated testing, and works well with Truffle. Check out it's README and documentation for more information.

Note that I *did* emphasize that it's separate, hopefully helping to sell `truffle develop` as being integrated, since users are really liking not having to download and install another application.

Reply:
Thanks for this. I wasn't very clear on how we wanted to sell TestRPC, or how we wanted to differentiate it from Truffle Develop.